build/build-gh-pages.js

```diff
@@ -1,5 +1,30 @@
 const fs = require("fs").promises;
 
+
+// This object exist only to format the HTML as designed
+// type: warning-type as stored as filename in design/svg-folder
+// title: title that will be desplayed in html
+// Order is based on this site: https://hjelp.yr.no/hc/no/articles/360014052634
+//
+// "type: wind" will look for files named "icon-warning-wind-${yellow/orange/red}.svg"
+const warningTypes = [
+  {type: 'wind', title: 'Gust/Wind'},
```

Comment:
Adding a note here since I can't comment on the SVG files.

I don't think we should move any of the SVG files to a deprecated folder. I don't really trust that we will never get a red forest fire event type any more, for example.

I think we should still generate all the icons, but this warningTypes array could include details about which colors to show per event type instead so we will still generate red forest fire, but the html page won't list it.